src/_data/starters/11ty-portfolio-starter

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://github.com/aadityavaze/11ty-portfolio-starter",
```

Comment:
https://github.com/aadityavaze/11ty-portfolio-starter seems to 404 for me.
Is that a private repo?

Reply:
Yes! I just changed it to public